frontend/resources/translations/hin.json

```diff
@@ -0,0 +1,744 @@
+        "group": "समूह",
+        "addFilter": "फ़िल्टर जोड़ें",
+        "deleteFilter": "फ़िल्टर हटाएँ",
+        "filterBy": "फ़िल्टरबाय...",
+        "typeAValue": "एक मान टाइप करें...",
+        "layout": "लेआउट",
+        "databaseLayout": "लेआउट"
+      },
+      "textFilter": {
+        "contains": "शामिल है",
+        "doesNotContain": "इसमें शामिल नहीं है",
+        "endsWith": "समाप्त होता है",
+        "startWith": "से प्रारंभ होता है",
+        "is": "है",
+        "isNot": "नहीं है",
+        "isEmpty": "खाली है",
+        "isNotEmpty": "खाली नहीं है",
+        "choicechipPrefix": {
+          "isNot": "नहीं",
+          "startWith": "से प्रारंभ होता है",
+          "endWith": "के साथ समाप्त होता है",
+          "isEmpty": "खाली है",
+          "isNotEmpty": "खाली नहीं है"
+        }
+      },
+      "checkboxFilter": {
+        "isChecked": "चेक किया गया",
+        "isUnchecked": "अनचेक किया हुआ",
+        "choicechipPrefix": {
+          "is": "है"
+        }
+      },
+      "checklistFilter": {
+        "isComplete": "पूर्ण है",
+        "isIncomplted": "अपूर्ण है"
+      },
+      "singleSelectOptionFilter": {
+        "is": "है",
+        "isNot": "नहीं है",
+        "isEmpty": "खाली है",
+        "isNotEmpty": "खाली नहीं है"
+      },
+      "multiSelectOptionFilter": {
+        "contains": "शामिल है",
+        "doesNotContain": "इसमें शामिल नहीं है",
+        "isEmpty": "खाली है",
+        "isNotEmpty": "खाली नहीं है"
+      },
+      "field": {
+        "hide": "छिपाएँ",
+        "insertLeft": "बायाँ सम्मिलित करें",
+        "insertRight": "दाएँ सम्मिलित करें",
+        "duplicate": "डुप्लिकेट",
+        "delete": "हटाएं",
+        "textFieldName": "टेक्स्ट",
+        "checkboxFieldName": "चेकबॉक्स",
+        "dateFieldName": "दिनांक",
+        "updatedAtFieldName": "अंतिम संशोधित समय",
+        "createdAtFieldName": "समय बनाया गया",
+        "numberFieldName": "नंबर",
+        "singleSelectFieldName": "चुनें",
+        "multiSelectFieldName": "मल्टीसेलेक्ट",
+        "urlFieldName": "URL",
+        "checklistFieldName": "चेकलिस्ट",
+        "numberFormat": "संख्या प्रारूप",
+        "dateFormat": "दिनांक प्रारूप",
+        "includeTime": "समय शामिल करें",
+        "isRange": "अंतिम तिथि",
+        "dateFormatFriendly": "माह दिन, वर्ष",
+        "dateFormatISO": "वर्ष-महीना-दिन",
+        "dateFormatLocal": "महीना/दिन/वर्ष",
+        "dateFormatUS": "वर्ष/महीना/दिन",
+        "dateFormatDayMonthYear": "दिन/माह/वर्ष",
+        "timeFormat": "समय प्रारूप",
+        "invalidTimeFormat": "अमान्य प्रारूप",
+        "timeFormatTwelveHour": "12 घंटा",
+        "timeFormatTwentyFourHour": "24 घंटे",
+        "clearDate": "तिथि साफ़ करें",
+        "addSelectOption": "एक विकल्प जोड़ें",
+        "optionTitle": "विकल्प",
+        "addOption": "विकल्प जोड़ें",
+        "editProperty": "संपत्ति संपादित करें",
+        "newProperty": "नई संपत्ति",
+        "deleteFieldPromptMessage": "क्या आप निश्चित हैं? यह संपत्ति हटा दी जाएगी",
```

Comment:
read them after you translate, try to think about the context you are translating them for.